13-transport.Rmd

```diff
@@ -613,15 +612,14 @@ route_network_scenario = overline(routes_short_scenario, attrib = "bicycle")
 
 The outputs of the two preceding code chunks are summarized in Figure \@ref(fig:rnetvis) below.
 
-```{r rnetvis, out.width="49%", fig.show='hold', fig.cap="Illustration of the % of car trips switching to cycling as a function of distance (left) and route network level results of this function (right).", echo=FALSE}
+```{r rnetvis, out.width="49%", fig.show='hold', fig.cap="Illustration of the % of car trips switching to cycling as a function of distance (left) and route network level results of this function (right).", echo=FALSE, fig.height=6}
 routes_short_scenario |> 
   ggplot() +
   geom_line(aes(distance / 1000, uptake)) +
-  xlab("Route distance (km)") +
-  ylab("Percent trips switching from driving to cycling") +
+  labs(x = "Route distance (km)", y = NULL, title = "Percent trips switching from driving to cycling") +
   scale_y_continuous(labels = scales::percent)
 tm_shape(route_network_scenario) +
-  tm_lines(lwd = "bicycle", scale = 9, title.lwd = "N. bike trips per day\n(modeled, one direction)")
+  tm_lines(lwd = "bicycle", scale = 9, title.lwd = "No. bike trips per day\n(modeled, one direction)")
```

Comment:
I vote Number of bike trips, to be explicit.

Reply:
No. is fine by me (better than N.); If you prefer "Number" you need to change it in several figures in this chapter.